include/intx/intx.hpp

```diff
@@ -33,7 +33,7 @@ struct uint
 
     constexpr uint() noexcept = default;
 
-    constexpr uint(half_type hi, half_type lo) noexcept : lo(lo), hi(hi) {}
+    constexpr uint(half_type high, half_type low) noexcept : lo(low), hi(high) {}
```

Comment:
Why not `_hi` and _lo`?

Reply:
Looks horrible in documentation and any IDE assistant.

Reply:
I don't mind was just curious as that is a recommendation in many places/languages nowadays.

Reply:
I try to avoid additional rules just to have less rules. I believe the `_` prefix was mostly required in cpp-ethereum.

Reply:
Solidity uses it too and now Rust as well.

Reply:
cpp-ethereum -> solidity.